selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/DashboardWorkspace.java

```diff
@@ -936,4 +943,45 @@ public void selectSample(String name) {
                 By.xpath(format(Locators.SAMPLE_CHECKBOX_XPATH, name))))
         .click();
   }
+
+  public void deleteAllWorkspaces(List<String> workspaces) {
+    workspaces.forEach(
+        wsName -> {
+          seleniumWebDriver.get(getDashboardWorkspaceUrl());
+          waitToolbarTitleName(WORKSPACE_TOOLBAR_TITLE);
+          waitListWorkspacesOnDashboard();
+          selectWorkspaceItemName(wsName);
+          waitToolbarTitleName(Arrays.asList(wsName.split("/")).get(1));
+          clickOnDeleteWorkspace();
+          clickOnDeleteDialogButton();
+          waitToolbarTitleName(WORKSPACE_TOOLBAR_TITLE);
```

Comment:
We need to ensure that workspace was completely removed. Method should wait on notification popup at the top right corner after workspace has been successfully removed.